sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1414,7 +1414,7 @@ class DataSourceV2SQLSuite
   }
 
   test("tableCreation: duplicate column names in the table definition") {
-    val errorMsg = "Found duplicate column(s) in the table definition of t"
+    val errorMsg = "Found duplicate column(s) in the table definition"
```

Comment:
if you look at the test, `t` refers to `spark_catalog.default.t`, `testcat.t` refers to `testcat.t`, the namespace is different, so the error message is also slightly different.